tests/stdlib/ttimes.nim

```diff
@@ -345,20 +345,24 @@ suite "ttimes":
   test "adding/subtracting TimeInterval":
     # add/subtract TimeIntervals and Time/TimeInfo
     let now = getTime().utc
+    let isSpecial = now.isLeapDay
     check now + convert(Seconds, Nanoseconds, 1).nanoseconds == now + 1.seconds
     check now + 1.weeks == now + 7.days
     check now - 1.seconds == now - 3.seconds + 2.seconds
     check now + 65.seconds == now + 1.minutes + 5.seconds
     check now + 60.minutes == now + 1.hours
     check now + 24.hours == now + 1.days
-    check now + 13.months == now + 1.years + 1.months
+    if not isSpecial:
```

Comment:
Just write `now.isLeapDay` directly

Reply:
less DRY so not a clear improvement

Reply:
Lol, what? And introducing a new variable that literally does nothing except for obfuscating the intent is more DRY?
Anyways doesn't matter, its just a test anyways :)